src/modules/chrome/helperMenu/HelperMenu.svelte

```diff
@@ -0,0 +1,122 @@
+<script>
+  import { onMount } from 'svelte';
+  import sendEvent from '../../analytics/sendEvent';
+  import draggable from '../../common/draggable';
+  import isFunction from '../../common/isFunction';
+  import querySelector from '../../common/querySelector';
+  import Modal from '../../modal/Modal.svelte';
+  import { playerIdUrl } from '../../support/constants';
+  import getValue from '../../system/getValue';
+  import functionLookup from './functionLookup';
+
+  const isFixed = getValue('keepHelperMenuOnScreen');
+  const isDraggable = getValue('draggableHelperMenu');
+
+  onMount(() => {
+    if (isDraggable) draggable(querySelector('#helperMenu'));
+  });
+
+  let showMenu = false;
+  function toggle() {
+    showMenu = !showMenu;
+  }
+
+  function sendHelperEvent(name) {
+    sendEvent('helperMenu', name);
+  }
+
+  function callModalFunction(name, fn) {
+    if (isFunction(fn)) {
+      sendHelperEvent(name);
+      fn();
+      toggle();
+    }
+  }
+</script>
+<button
+  class:helperMenuFixed={ isFixed }
+  class:helperMenuMove={ isDraggable }
+  type="button"
+  id="toggle"
+  on:click={ toggle }>
+  Helper Menu
+</button>
+<Modal visible={ showMenu } on:close={ toggle }>
```

Comment:
is there a way to make the modal appear underneath the HelperMenu toggle button?